packages/desktopjs/src/propertymapping.ts

```diff
@@ -7,12 +7,14 @@ export class PropertyMap {
     [id: string]: { target: string, convert?: (value: any, from: any, to: any) => any };
 }
 
+// eslint-disable-next-line @typescript-eslint/no-namespace
 export namespace ObjectTransform {
```

Comment:
I suspect we will want to move away from using namespaces long term but will leave this as is for now

Reply:
Can we just switch to using objects?
```
export const ObjectTransform = { 
...
}
```
It's going to be the same code at run-time, not sure if there are any differences at compile-time.

Reply:
Should be an easy test as the other packages reference this.  Let's try it out after this is all merged.